item-spec/common-metadata.md

```diff
@@ -29,17 +29,21 @@ Descriptive fields to give a basic overview of a STAC Item.
 
 ## Date and Time
 
+- [JSON Schema](json-schema/datetime.json)
+
 Fields to provide additional temporal information such as ranges with a start and an end datetime stamp.
 
-### Date and Time Range
+| Field Name | Type         | Description |
+| ---------- | ------------ | ----------- |
+| datetime   | string\|null | See the [Item Spec Fields](item-spec.md#properties-object) for more information. |
```

Comment:
I think it's valuable to mention this here so that we have a full overview here. Otherwise you have one field in the Item Spec and all the other here, which is a bit confusing.

Reply:
+1